### ISSUE TITLE
First step moco

### ISSUE THREAD

**Issue by txw1997** [wvangansbeke/Unsupervised-Classification#41]
Hello, thanks for your excellent work! I use moco.py, but the Accuracy of top-50 nearest neighbors on train/val set is nan. Have you meet this? what can I do to avoid this?

**Comment by txw1997** [comment]
I found the problem because the dataset is empty. I want to know how you set up the validation set. Is it the same class as the training set?

**Comment by wvangansbeke** [comment]
Hi @txw1997,

Which command are your running exactly for ImageNet? Which config file are you using?
Are you sure that your ImageNet dataset path is correctly set?

Normally you can simply change the `train_db_name` (or `val_db_name`) in the config file to `imagenet`. This will execute [this line](https://github.com/wvangansbeke/Unsupervised-Classification/blob/master/utils/common_config.py#L135). Maybe have a look at the length of the dataset object, i.e. `len(dataset)` as a sanity check.

**Comment by txw1997** [comment]
Thanks for apply!
use this command ----python simclr.py --config_env configs/env.yml --config_exp configs/pretext/moco_imagenet_50.yml
The validation set I use is not have classes, just 50000 images. I have the 1000classes imagenet training set, but I dont know which validation set should I use?

**Comment by wvangansbeke** [comment]
Hi @txw1997,

The command you used seems correct. What is the problem exactly? Because 50k images in the validation set (i.e. 50 images per class) is correct.

**Comment by txw1997** [comment]
Oh!! I got it. Sorry, I never used val set of imagenet before. I should put image into corresponding folder first.
Thanks you for your patience.
